### ISSUE TITLE
config file is missing the Drug_Repurposing_Path

### ISSUE THREAD

**Issue by janjoy** [BaranziniLab/KG_RAG#41]
Hi @karthiksoman,

I am trying to run the kg_rag/rag_based_generation/GPT/run_drug_repurposing.py script, but I encountered an issue. The config file is missing the Drug_Repurposing_Path, and I can't seem to locate the data (68 drug repurposing questions) for it either. Could you please let me know where I can find these?

Thank you for your help!

**Comment by karthiksoman** [comment]
@janjoy In our analysis, we chose not to include the drug repurposing example, as we thought it could potentially lead to diverging interpretations or unintended implications. As a result, we replaced this analysis in the latest version of our manuscript, which is why the drug repurposing data is not present in the repo. However, I’m happy to share the initial data we used, and I have attached it here for your reference.
[drug_repurposing_questions_v2.csv](https://github.com/user-attachments/files/17796298/drug_repurposing_questions_v2.csv)
